src/modules/create-market/actions/submit-new-market.js

```diff
@@ -39,22 +37,37 @@ export function submitNewMarket(newMarket, history) {
             eachOfSeries(Object.keys(newMarket.orderBook), (outcome, index, seriesCB) => {
               eachLimit(newMarket.orderBook[outcome], constants.PARALLEL_LIMIT, (order, orderCB) => {
                 const outcomeId = newMarket.type === CATEGORICAL ? index : 1 // NOTE -- Both Scalar + Binary only trade against one outcome, that of outcomeId 1
-                dispatch(updateTradesInProgress(marketId, outcomeId, order.type === BID ? BUY : SELL, order.quantity, order.price, null, (err, tradingActions) => {
-                  const tradeToExecute = {
-                    [outcomeId]: tradingActions,
-                  }
-
-                  if (tradeToExecute) {
-                    dispatch(placeTrade(marketId, outcomeId, tradeToExecute[outcomeId], null, (err) => {
-                      if (err) return console.error('ERROR: ', err)
-
-                      orderCB()
-                    }))
-                  }
-                }))
+                const orderType = order.type === BID ? 0 : 1
+                const numTicks = formattedNewMarket.tickSize ? (newMarket.scalarBigNum - newMarket.scalarSmallNum) / newMarket.tickSize : augur.constants.DEFAULT_NUM_TICKS[2]
+                const tradeCost = augur.trading.calculateTradeCost({
+                  displayPrice: order.price,
+                  displayAmount: order.quantity,
+                  sharesProvided: '0',
+                  numTicks,
+                  orderType,
+                  minDisplayPrice: newMarket.scalarSmallNum || 0,
+                  maxDisplayPrice: newMarket.scalarBigNum || 1,
+                })
+                const { onChainAmount, onChainPrice, cost } = tradeCost
+                augur.api.CreateOrder.publicCreateOrder({
+                  meta: loginAccount.meta,
+                  tx: { value: augur.utils.convertBigNumberToHexString(cost) },
+                  _type: orderType,
+                  _attoshares: augur.utils.convertBigNumberToHexString(onChainAmount),
+                  _displayPrice: augur.utils.convertBigNumberToHexString(onChainPrice),
+                  _market: marketId,
+                  _outcome: outcomeId,
+                  _tradeGroupId: augur.trading.generateTradeGroupId(),
+                  onSent: noop,
+                  onSuccess: (res) => {
+                    orderCB()
+                  },
+                  onFailed: (err) => {
+                    console.error('ERROR creating order in initial market liquidity: ', err)
+                  },
```

Comment:
Is orderCD() needed if publicCreateOrder fails? seems like it would be in waiting state.

Reply:
Yea I think you're right!. Weirdly that doesn't appear to be how the old behavior worked